.gitignore

```diff
@@ -1,13 +1,31 @@
+# Logs
+logs
+*.log
+yarn-debug.log*
+yarn-error.log*
+
 node_modules
-.DS_Store
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+!.vscode/settings.json
```

Comment:
Last nit, there is no need to store any VSCode related files since we do not recommend any specific IDE, you can use whatever you want, so let's ignore all editor files:
```suggestion
```